### ISSUE TITLE
Question about APP_DOMAIN and APP_PATH

### ISSUE THREAD

**Issue by rtm** [chemerisuk/cordova-plugin-firebase-dynamiclinks#44]
Excuse the possibly stupid question and/or posting it in the wrong place. The docs state

> Variables `APP_DOMAIN` and `APP_PATH` specify web URL where your app will start an activity to handle the link.

But I'm not familiar with the terms "activity" in the Cordova context, or what it means for my app to "start an activity". 

What do these variables actually mean and what are they used for? Is `APP_DOMAIN` supposed to be the `https://xxxxx.goo.gl` domain?

Also, what is the internal structure of the `data` parameter passed to the `onDynamicLink` callback?

**Comment by Bengejd** [comment]
> What do these variables actually mean and what are they used for? Is APP_DOMAIN supposed to be the https://xxxxx.goo.gl domain?

Yes, the APP_DOMAIN is supposed to be https://YOUR_DYNAMIC_LINK_CODE.app.goo.gl/

> Also, what is the internal structure of the data parameter passed to the onDynamicLink callback?
{"matchType":"Strong","deepLink":"https://DEEP_LINK_AUTH_URL_HERE"}

So you would pass the `deepLink` url to your `SigninWithEmailLink() ` method, and voila, you are signed in.

**Comment by The-Don-Himself** [comment]
@rtm thank you for asking this question and @Bengejd for giving a great answer, I was about to mess up by using my actual web app domain instead of firebase deeplink domain. Maybe an upgrade to the docs is in order?

**Comment by Bengejd** [comment]
Glad to help @The-Don-Himself !

**Comment by fdambrosio** [comment]
thanks @Bengejd  , I can suggest to upgrade doc  s
